Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderThumbnailView.cpp

```diff
@@ -334,6 +344,7 @@ namespace AzQtComponents
         config.topItemsHorizontalSpacing = 18;
         config.topItemsVerticalSpacing = 18;
         config.childrenItemsHorizontalSpacing = 7;
+        config.scrollSpeed = 45;
```

Comment:
`45` is an arbitrary magic number, maybe make a constant here with a name describing what `45` means

Reply:
Sure I'll add a comment, there are a ton of magic numbers as part of this struct, but I think that's on purpose since it's a custom abstract item layout